lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -16,27 +16,27 @@ class Form4142Processor
 
       def initialize(form_data:, submission_id: nil)
         @form = form_data
+        @submission = Form526Submission.find_by(id: submission_id)
         @pdf_path = generate_stamp_pdf
         @uuid = SecureRandom.uuid
         @request_body = {
           'document' => to_faraday_upload,
           'metadata' => generate_metadata
         }
-        @submission = Form526Submission.find_by(id: submission_id)
```

Comment:
This line needed to be moved up so that it was set before the `generate_stamp_pdf` method was called.